lib/commands/report.js

```diff
@@ -47,6 +47,12 @@ exports.builder = function (yargs) {
       default: false,
       type: 'boolean'
     })
+    .option('build-process-tree', {
```

Comment:
Sorry I didn't catch this earlier (not sure if it was here), but this doesn't make sense.  `nyc report` doesn't execute any processes so it can't build a process tree (it can only show it).  IMO it only makes sense to add this option to `lib/config-utils.js` for the 'main' command.

Reply:
Good point!  Either the processinfo is there or it isn't, but `nyc report` can't put it there.  Removed and pushed.